app/api/signup/views.py

```diff
@@ -0,0 +1,14 @@
+from fastapi import APIRouter
+from .schemas import Signup
+
+router = APIRouter()
+
+@router.get("/", response_model= Signup)
```

Comment:
Signup should be a POST endpoint with the `first_name` etc included in the request body.